x-pack/test/reporting_api_integration/reporting_and_security/network_policy.ts

```diff
@@ -40,7 +40,7 @@ export default function ({ getService }: FtrProviderContext) {
         filter(({ statusCode }) => statusCode === 500),
         map(({ message }) => message),
         first(),
-        timeout(15000)
+        timeout(120000)
```

Comment:
This change is not related to a test that is currently failing, but the value was still way too low and was on the cusp of being flaky.

One of the reasons being very low was acceptable in this test is that the use case it covers has bug which makes it fail early instead of going through the usual 3 attempts to complete a report.